pkg/api-server/testdata/inspect_meshtrafficpermission.json

```diff
@@ -0,0 +1,19 @@
+{
+ "total": 1,
+ "items": [
+  {
+   "kind": "SidecarDataplane",
+   "dataplane": {
+    "mesh": "mesh-1",
+    "name": "backend-1"
+   },
+   "attachments": [
+    {
+     "type": "dataplane",
+     "name": "",
+     "service": ""
```

Comment:
This is odd too

Reply:
same as https://github.com/kumahq/kuma/pull/4977#discussion_r968126477